src/components/resource/components/resource-information/resource-information.js

```diff
@@ -0,0 +1,169 @@
+            </InternalLink>
+          </KeyValue>
+
+          {model.title.edition && (
+            <KeyValue label={<FormattedMessage id="ui-eholdings.label.edition" />}>
+              <div data-test-eholdings-resource-show-edition>
+                {model.title.edition}
+              </div>
+            </KeyValue>
+          )}
+
+          <ContributorsList data={model.title.contributors} />
+
+          {model.title.publisherName && (
+            <KeyValue label={<FormattedMessage id="ui-eholdings.label.publisher" />}>
+              <div data-test-eholdings-resource-show-publisher-name>
+                {model.title.publisherName}
+              </div>
+            </KeyValue>
+          )}
+
+          {model.title.publicationType && (
+            <KeyValue label={<FormattedMessage id="ui-eholdings.label.publicationType" />}>
+              <div data-test-eholdings-resource-show-publication-type>
+                {model.title.publicationType}
+              </div>
+            </KeyValue>
+          )}
+
+          <IdentifiersList data={model.title.identifiers} />
+
+          {model.title.subjects.length > 0 && (
+            <KeyValue label={<FormattedMessage id="ui-eholdings.label.subjects" />}>
+              <div data-test-eholdings-resource-show-subjects-list>
+                {model.title.subjects.map(subjectObj => subjectObj.subject).join('; ')}
+              </div>
+            </KeyValue>
+          )}
+
+          <KeyValue label={<FormattedMessage id="ui-eholdings.label.peerReviewed" />}>
+            <div
+              data-test-eholdings-peer-reviewed-field
+              data-testid="peer-reviewed-field"
+            >
+              {
+                model.title.isPeerReviewed
+                  ? (<FormattedMessage id="ui-eholdings.yes" />)
+                  : (<FormattedMessage id="ui-eholdings.no" />)
+              }
+            </div>
+          </KeyValue>
+
+          <KeyValue label={<FormattedMessage id="ui-eholdings.label.titleType" />}>
+            <div
+              data-test-eholdings-package-details-type
+              data-testid="package-details-type"
+            >
+              {
+                model.title.isTitleCustom
+                  ? (<FormattedMessage id="ui-eholdings.custom" />)
+                  : (<FormattedMessage id="ui-eholdings.managed" />)
+              }
+            </div>
+          </KeyValue>
+
+          {model.title.description && (
+            <KeyValue label={<FormattedMessage id="ui-eholdings.label.description" />}>
+              <div data-test-eholdings-description-field>
+                {model.title.description}
+              </div>
+            </KeyValue>
+          )}
+        </div>
+        <div>
+          <KeyValue label={<FormattedMessage id="ui-eholdings.label.package" />}>
+            <div data-test-eholdings-resource-show-package-name>
+              <InternalLink
+                to={`/eholdings/packages/${model.packageId}`}
+              >
```

Comment:
Please, write single attribute on the same line with tag name
```suggestion
              <InternalLink to={`/eholdings/packages/${model.packageId}`} >
```